### DISCUSSION TITLE
icons example does not work

### DISCUSSION THREAD

**Discussion by aborruso** [squidfunk/mkdocs-material#2846] [General]
Hi,
in the [documentation](https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/?h=emo#using-icons)  the third example is not rendered: it remains ``:octicons-octoface-24:``.

![image](https://user-images.githubusercontent.com/30607/126072802-68dc01b3-2663-4195-bdf0-9e7e6a7c86d6.png)

**Comment by facelessuser** [comment]
It's possible that icon was renamed or removed.

**Reply by squidfunk** [reply]
Yes, it seems it was removed. I switched to another icon in c843fc742.

**Reply by aborruso** [reply]
Thank you
